Python/optimizer.c

```diff
@@ -543,11 +543,23 @@ translate_bytecode_to_trace(
                 int counter = instr[1].cache;
                 int bitcount = _Py_popcount32(counter);
                 int jump_likely = bitcount > 8;
+                if (jump_likely) {
+                    confidence *= bitcount / 16.0;
```

Comment:
Division is notoriously slow and beware of implicit casts to `double`, so can you make this `bitcount * (float)(1/16.0)`.